File: test/tutorial/api_changes_from_dnf4/configure_base.cpp
// Get or set configuration options.
auto & config = base.get_config();
config.get_skip_unavailable_option().set(true);
std::cout << config.get_skip_unavailable_option().get_value() << std::endl;

// Get or set a value of a variable, e.g.:
auto vars = base.get_vars();
vars->set("releasever", "42");
std::cout << vars->get_value("releasever") << std::endl;

File: test/python3/libdnf5/tutorial/api_changes_from_dnf4/configure_base.py
# Get or set configuration options.
config = base.get_config()
config.skip_unavailable = True
print(config.skip_unavailable)

# Get or set values of variables.
vars = base.get_vars()
vars.set("releasever", "42")
print(vars.get_value("releasever"))
